Tests/ScenarioTester/.editorconfig

```diff
@@ -0,0 +1,203 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
+############################### 
+# Core EditorConfig Options   # 
```

Comment:
Great that you added .editorconfig. Is this editorconfig the same as the version under: https://github.com/microsoft/data-accelerator/blob/master/Services/.editorconfig?

Reply:
yes, it is the same.